internal/app/api/types/pagesort.go

```diff
@@ -0,0 +1,37 @@
+// Pagination and Sorting
+package types
+
+import "strconv"
+
+type PagingNSorting struct {
+	Size int `json:"size" default:"100"`
+	Page int `json:"page" default:"1"`
+}
+
+func (ps *PagingNSorting) Init(page, size string) error {
+
+	if size == "" && page == "" {
```

Comment:
Have you tested this case?

Reply:
this gets all user
![image](https://user-images.githubusercontent.com/48717211/124688764-8f798800-df01-11eb-8b1c-5c44a2d607fe.png)
or
![image](https://user-images.githubusercontent.com/48717211/124688821-a029fe00-df01-11eb-917f-01eeac95bf22.png)